packages/generator/src/service/index-file.ts

```diff
@@ -21,11 +21,9 @@ export function indexFile(service: VdmServiceMetadata): SourceFileStructure {
         exportStatement(complexType.typeName)
       ),
       ...service.enumTypes.map(enumType => exportStatement(enumType.typeName)),
-      ...(service.functionImports && service.functionImports.length
-        ? [exportStatement('function-imports')]
-        : []),
-      ...(service.actionImports && service.actionImports.length
-        ? [exportStatement('action-imports')]
+      ...((service.functionImports && service.functionImports.length) ||
```

Comment:
[pp] you could use conditional access `service.functionImports?.length` to shorten it.

Reply:
Good idea, and I think this part of the code could be updated further because by definition `service.functionImports` can't be undefined or null. I'll use the conditional access on `service.actionImports`.